deploy/cert-renew.yml

```diff
@@ -6,6 +6,20 @@
         that:
           - cert_dir is defined
 
+    - name: 'Copy manual auth script'
+      copy:
+          src: '{{ item }}'
+          dest: '{{ dst_dir | mandatory }}/domain-auth/'
+          with_fileglob:
+            - '{{ cdn_dir | mandatory }}/*.sh'
+        become: yes
```

Comment:
You must reduce the indentation for with_fileglob. See https://docs.ansible.com/ansible/latest/collections/ansible/builtin/fileglob_lookup.html#examples

Reply:
The same thing goes for `become`